stdlib/public/stubs/Random.cpp

```diff
@@ -65,16 +65,18 @@ void swift_stdlib_random(void *buf, __swift_size_t nbytes) {
 
 SWIFT_RUNTIME_STDLIB_API
 void swift_stdlib_random(void *buf, __swift_size_t nbytes) {
-  if (nbytes > ULONG_MAX) {
-    fatalError(0, "Fatal error: %zd exceeds ULONG_MAX\n", nbytes);
-  }
+  while (nbytes > 0) {
+    auto actual_nbytes = std::min(nbytes, (__swift_size_t)ULONG_MAX);
```

Comment:
It would be nice to use `static_cast` as this is C++.

```suggestion
    auto actual_nbytes = std::min(nbytes, static_cast<__swift_size_t>(ULONG_MAX));
```

Reply:
Good question. AFAIK the unit tests for random number generation are not conditionalized by platform, are they?